mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -2315,7 +2315,7 @@ public void onClick(final DialogInterface dialog, final int which) {
                 final int limit = query != null ? query.getInt("maxResults", 5) : 5;
                 final BrowserDB sdb = BrowserDB.from(getProfile());
                 final ContentResolver scr = getContentResolver();
-                final Cursor scu = sdb.getHistoryForQuery(scr, text, limit);
+                final Cursor scu = sdb.getRankedHistoryForQuery(scr, text, limit);
```

Comment:
```Java
/* Cliqz Start */
<your code here>
/* Cliqz End */
```

Reply:
I just renamed the custom cliqz `getHistoryForQuery` to `getRankedHistoryForQuery`. I marked `getRankedHistoryForQuery` in a `Cliqz` block in the abstract `BrowserDB` file. 

The whole block here `case "Search:GetHistory"` is wrapped in a `Cliqz` block anyway.